package/piksi_ins/piksi_ins.mk

```diff
@@ -5,22 +5,25 @@
 #############################################################
 
 ifeq    ($(BR2_HAS_PIKSI_INS),y)
```

Comment:
I went around in circles trying to figure out where this environment variable is set;

I found it a little confusing that we always include docker.mk even when doing a native build.  Not a big deal, but still confusing.  I understand this logic is to nullify the project if you don't actually have access to the ins repo, however.  

Should we have a warning though printed if BR2_HAS_PIKSI_INS isn't true?

Reply:
See 44f1abf for fixes.